services/user.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth import get_user_model
+
+User = get_user_model

```

Comment:
There is a mistake in assigning the `User` model. It should be `User = get_user_model()` with parentheses to correctly call the function and retrieve the user model.